app/src/main/java/com/willowtree/vocable/settings/EditCategoriesListFragment.kt

```diff
@@ -165,10 +165,28 @@ class EditCategoriesListFragment : BaseFragment<FragmentEditCategoriesListBindin
                 editCategorySelectButton.isEnabled = false
             }
 
-            editCategorySelectButton.action = {
-                val action = EditCategoriesFragmentDirections.actionEditCategoriesFragmentToEditCategoryOptionsFragment(category)
-                findNavController().navigate(action)
-                editCategoriesViewModel.onCategorySelected(category)
+            editCategorySelectButton.apply {
+                isEnabled = category.isUserGenerated
+
+                editCategorySelectButton.action = {
+                    val action = EditCategoriesFragmentDirections.actionEditCategoriesFragmentToEditCategoryOptionsFragment(category)
+                    findNavController().navigate(action)
+                    editCategoriesViewModel.onCategorySelected(category)
+                }
+            }
+        }
+
+        editButtonBinding.showHideCategoryButton?.apply {
+            if (!category.hidden) {
```

Comment:
We've been using drawable selectors to handle changing the drawable based on button states. So you can set the image in xml and then the drawable selector will handle toggling the image based on whether we disabled the button above. Take a look at things like `presets_action_button_icon` to see how we've been handling that.